.gitignore

```diff
@@ -20,3 +20,6 @@
 dist/**/*
 degitx
 degitx-gitaly
+/simulator/target/maven-status/maven-compiler-plugin/compile/default-cli/inputFiles.lst
```

Comment:
@stepanvalyavskiy I suppose we can ignore everything in `/simulator/target/**` dir, isn't it?